js/adapt-stacklist.js

```diff
@@ -74,6 +74,7 @@ define(function(require) {
                 duration: this.TRANSITION_TIME,
                 complete: function() {
 			        $item.addClass("show");
+					$item.a11y_focus();
```

Comment:
Could you please change the indentation to 4 spaces :)

Also, not so bothered, but this could potentially be simplified to `$item.addClass("show").a11y_focus();` thanks to jQuery's method chaining :)